apps/tangle-cloud/src/pages/instances/Instances/UpdateBlueprintModel/RejectConfirmationModal.tsx

```diff
@@ -39,7 +39,7 @@ function RejectConfirmationModal({
       description="Are you sure you want to reject this blueprint?"
     >
       <ModalHeader onClose={onClose} className="pb-4">
-        Service Request #{selectedRequest?.requestId}
+        Service Request #{selectedRequest?.requestId?.toString()}
```

Comment:
If `selectedRequest` or `requestId` is `undefined` or `null`, the expression evaluates to `undefined`, and the rendered text will be:

Service Request # (with nothing after the #)